src/accordion.js

```diff
@@ -0,0 +1,11 @@
+export default function() {
+  var bar = $(`.accordion__bar`);
+  var list = $(`.accordion__bar-list`);
+
+  bar.on('click', function(ev) {
+    var siblings = list.nextSibling;
+    $(this).siblings().slideToggle();
+    $(this).find(siblings).slideUp();
```

Comment:
Check to see what `siblings` and `$(this).find(siblings)` will select using a `debugger;` statement. The accordion is working on the actual click, but it isn't toggling the other ones. Draw out because when a title bar is clicked, find the OTHER `accordion__drop-down` then go inside of that and get the `accordion__bar-list`s and slide those up.